docs/DSP/GameCube_DSP_Users_Manual/GameCube_DSP_Users_Manual.tex

```diff
@@ -0,0 +1,3737 @@
+\end{tabular}
+\end{table}
+
+\pagebreak{}
+
+\chapter{Registers}
+
+\section{Register names}
+
+The DSP has 32 16-bit registers, although their individual purpose and their function differ from register to register.
+
+\begin{table}[htb]
+\centering
+\begin{tabular}{|l|l|l|l|}
+\hline
+                &                  &                      &                       \\ \hline
+\Register{\$0}  & \Register{\$r00} & \Register{\$ar0}     & Addressing register 0 \\ \hline
+\Register{\$1}  & \Register{\$r01} & \Register{\$ar1}     &                       \\ \hline
+\Register{\$2}  & \Register{\$r02} & \Register{\$ar2}     &                       \\ \hline
+\Register{\$3}  & \Register{\$r03} & \Register{\$ar3}     &                       \\ \hline
+\Register{\$4}  & \Register{\$r04} & \Register{\$ix0}     &                       \\ \hline
+\Register{\$5}  & \Register{\$r05} & \Register{\$ix1}     &                       \\ \hline
+\Register{\$6}  & \Register{\$r06} & \Register{\$ix2}     &                       \\ \hline
+\Register{\$7}  & \Register{\$r07} & \Register{\$ix3}     &                       \\ \hline
+\Register{\$8}  & \Register{\$r08} &                      &                       \\ \hline
+\Register{\$9}  & \Register{\$r09} &                      &                       \\ \hline
+\Register{\$10} & \Register{\$r0A} &                      &                       \\ \hline
+\Register{\$11} & \Register{\$r0B} &                      &                       \\ \hline
+\Register{\$12} & \Register{\$r0C} & \Register{\$st0}     &                       \\ \hline
+\Register{\$13} & \Register{\$r0D} & \Register{\$st1}     &                       \\ \hline
+\Register{\$14} & \Register{\$r0E} & \Register{\$st2}     &                       \\ \hline
+\Register{\$15} & \Register{\$r0F} & \Register{\$st3}     &                       \\ \hline
+\Register{\$16} & \Register{\$r10} & \Register{\$ac0.h}   &                       \\ \hline
+\Register{\$17} & \Register{\$r11} & \Register{\$ac1.h}   &                       \\ \hline
+\Register{\$18} & \Register{\$r12} & \Register{\$config}  &                       \\ \hline
+\Register{\$19} & \Register{\$r13} & \Register{\$sr}      &                       \\ \hline
+\Register{\$20} & \Register{\$r14} & \Register{\$prod.l}  &                       \\ \hline
+\Register{\$21} & \Register{\$r15} & \Register{\$prod.m1} &                       \\ \hline
+\Register{\$22} & \Register{\$r16} & \Register{\$prod.h}  &                       \\ \hline
+\Register{\$23} & \Register{\$r17} & \Register{\$prod.m2} &                       \\ \hline
+\Register{\$24} & \Register{\$r18} & \Register{\$ax0.l}   &                       \\ \hline
+\Register{\$25} & \Register{\$r19} & \Register{\$ax1.1}   &                       \\ \hline
+\Register{\$26} & \Register{\$r1A} & \Register{\$ax1.h}   &                       \\ \hline
+\Register{\$27} & \Register{\$r1B} & \Register{\$ax1.h}   &                       \\ \hline
```

Comment:
The duplicate `$ax1.h` seems to be a typo that's also present in the original. I'm guessing the first one should be `$ax0.h`.